README.md

```diff
@@ -50,4 +54,6 @@ so that you can use the OpenAI API SDKs with keyless (Entra) authentication. By
     python example.py
     ```
 
-    This will use the OpenAI API SDK to make a request to the OpenAI API and print the response.
+    This will use the OpenAI API SDK to make a request to the OpenAI API and print the response. 
```

Comment:
I'm also not sure if it was just me but after deployment I got a connection error. This error stopped after a few minutes and I assumed this was because the deployment might need some time to start working. Is this correct to assume? It might just be the lag with the specific location that was the issue like you mentioned above? 

Happy to delete this if it's not correct.

Reply:
I haven't experienced it, but it is possible. Are you sure it wasn't that you were missing the .env, and then later did `azd env get-values > .env`? 

One tip is to enable DEBUG level logging in your Python file, as then you'll see all the HTTP request URLs that go over the wire. I could even do that in the sample. Or just put an explicit check that the environment variables aren't empty.

Reply:
Okay so I just ran through the steps as fast as I could, and couldn't replicate a connection error. And I verified a connection error when .env is empty. So I've added a check for necessary env variables to the sample, committed to main.

Reply:
Can you remove this section for now, since I'm not sure there's actually a deployment lag?

Reply:
Yes! That makes sense! I'll delete this line wording then. 
Having the note on debugging would be great! +1 on including it in the demo

Reply:
I've added logging but default'ed it to WARNING with a note to change to DEBUG to see SDK logs. It's super verbose so you only want it if debugging errors.

Reply:
Great! I've removed the extra line here about the lag! That makes sense to me!